Source/Blazorise/README.md

```diff
@@ -0,0 +1,17 @@
+﻿## Build external libraries
+
+External libraries must be build as a valid JS module to be consumed by Blazorise.
+
+In this section we will describe how to build external libraries that are used in Blazorise, step by step. 
+
+### Autonumeric
+
+GitHub: https://github.com/autoNumeric/autoNumeric/
+
+1. clone the project to your local machine
```

Comment:
Most libraries already do releases with the proper artifacts that can just be copied? Is this a mandatory process?

Reply:
Not all libraries. Some do not have any JS module support, or the JS module is not valid to be imported for our use. So, for those use cases, we must have a process that is working. Because every library has their own specificities, I want to have it all written down step by step. Otherwise, we will need to relearn it every time. Yesterday, I lost more time doing the rebuild than the actual work. Just because I forgot how I did it last time.